templates/security/certifications.html

```diff
@@ -3,202 +3,203 @@
-          alt="",
-          height="300",
-          width="250",
-          hi_def=True,
-          loading="lazy",
-          attrs={"class": "u-hide--small u-hide--medium"}
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-<section class="p-strip is-bordered u-no-padding--top">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <p>Canonical's security certification offerings are now available to all Ubuntu Advantage for Infrastructure and Ubuntu Pro customers on AWS and Microsoft Azure.</p>
-      <p>Launch Ubuntu Pro 18.04 LTS on <a class="p-link--external" href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic?tab=Overview">Azure</a> and <a class="p-link--external" href="https://aws.amazon.com/marketplace/pp/B0821T9RL2">AWS</a></p>
-      <p><a href="/support">Learn more about Ubuntu Advantage&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 u-hide--small u-align--center u-vertically-center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
-          alt="",
-          height="51",
-          width="180",
-          hi_def=True,
-          loading="lazy",
-          attrs={"class": ""}
-        ) | safe
-      }}
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg",
-          alt="",
-          height="60",
-          width="100",
-          hi_def=True,
-          loading="lazy",
-          attrs={"class": ""}
-        ) | safe
-      }}
-    </div>
-  </div>
+ <div class="row u-equal-height u-vertically-center">
+   <div class="col-8">
+     <h1>Security Certifications</h1>
+     <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
+     <p>Regulated and high security environments have complex requirements as they are designed to tackle many threats, known and unknown. Whatever path you have chosen in navigating compliance with a cybersecurity framework, such as the ones defined by ISO 27000, NIST, PCI or CIS Controls, Canonical provides you with a secure and easy to use operating system, that enables your compliance and reduces operational risk, whether on-premise or on a public cloud. We, at Canonical, are transparent on the security features we enable, provide access and automation to Industry accepted hardening and compliance profiles, such as CIS and DISA-STIG as well as present 3rd party attestation and certifications like FIPS 140-2 and Common Criteria.</p>
+     <p><a class="p-button--positive js-invoke-modal" href="/support/contact-us?product=support-overview">Contact us</a></p>
+   </div>
+   <div class="col-4 u-hide--small u-align--center">
+     {{
+       image(
+         url="https://assets.ubuntu.com/v1/7953a068-security-1.svg",
+         alt="",
+         height="300",
+         width="250",
+         hi_def=True,
+         loading="lazy",
```

Comment:
```suggestion
         loading="auto",
```
Set images in the header to auto, everything below to lazy